packages/wallets/package.json

```diff
@@ -331,30 +331,27 @@
     "@safe-global/safe-ethers-lib": "^1.9.2",
     "@thirdweb-dev/chains": "workspace:*",
     "@thirdweb-dev/sdk": "workspace:*",
-    "@walletconnect/client": "^1.8.0",
-    "@walletconnect/core": "2.7.3",
-    "@walletconnect/ethereum-provider": "2.6.0",
-    "@walletconnect/jsonrpc-http-connection": "^1.0.0",
-    "@walletconnect/jsonrpc-provider": "^1.0.3",
-    "@walletconnect/jsonrpc-types": "^1.0.1",
+    "@walletconnect/core": "^2.8.4",
+    "@walletconnect/ethereum-provider": "^2.8.4",
     "@walletconnect/jsonrpc-utils": "^1.0.3",
-    "@walletconnect/legacy-client": "^2.0.0-rc.0",
-    "@walletconnect/legacy-modal": "^2.0.0-rc.0",
-    "@walletconnect/legacy-types": "^2.0.0-rc.0",
-    "@walletconnect/legacy-utils": "^2.0.0-rc.0",
-    "@walletconnect/types": "2.7.3",
-    "@walletconnect/utils": "2.7.3",
-    "@walletconnect/web3wallet": "^1.7.1",
-    "@web3modal/standalone": "^2.4.1",
+    "@walletconnect/modal": "^2.5.4",
+    "@walletconnect/types": "^2.8.4",
+    "@walletconnect/utils": "^2.8.4",
+    "@walletconnect/web3wallet": "^1.8.1",
     "buffer": "^6.0.3",
     "cross-fetch": "^3.1.5",
     "crypto-js": "^4.1.1",
-    "eip1193-provider": "^1.0.1",
     "eth-provider": "^0.13.6",
     "eventemitter3": "^5.0.0",
     "magic-sdk": "^13.6.2",
     "web3-core": "1.5.2"
   },
+  "resolutions": {
```

Comment:
resolutions on the package-level don't do anything with `pnpm` @iketw @MananTank - if you *need* these resolutions you will have to put them in the root package.json

Reply:
(if you update your pnpm version to latest it will also warn you about this)